from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+#  Way to workaround while it is not in pip
+#  1. clone estimators/rlos2021_cfe, python3 setup.py install
+#  2. copy estimators folder from rlos2021_cfe branch of estimators repo
+# import estimators 
+
+def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict: 
+# list of estimations for every estimator from config
+    # bucket : df 
+    # config : {} of results
+#   ..{} -> df 
+    return {}
+
+def estimate_bucket_only_ips(bucket: pd.DataFrame, config: dict) -> dict: 
```

Comment:
not sure why do we need this method. This file should contain (in this iteration) only estimate_bucket method with generic logic about doing estimation based on the config and the data from the bucket.
And the test_estimate should have different scenarios of calling this function. And that could be the right place for having method like test_ips_only where you could call estimate_bucket with only ips estimator in the config.